client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -178,7 +183,13 @@ class ThemeEnhancements extends Component {
 					) }
 					link="https://jetpack.com/support/mobile-theme/"
 				/>
-
+				<FormLegend>{ translate( ' Mobile Theme' ) }</FormLegend>
```

Comment:
Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'Mobile Theme' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/34235-theme-enhancements)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).